substrate/frame/nfts/src/impl_nonfungibles.rs

```diff
@@ -320,6 +320,33 @@ impl<T: Config<I>, I: 'static> Mutate<<T as SystemConfig>::AccountId, ItemConfig
 		})
 	}
 
+	fn set_metadata(
```

Comment:
shall we name it as `set_item_metadata`, wdyt?

Reply:
Is calling `do_set_item_metadata` so for me makes sense to change the naming and use `set_item_metadata`